agent/xds/config.go

```diff
@@ -67,36 +67,67 @@ func ParseProxyConfig(m map[string]interface{}) (ProxyConfig, error) {
 	return cfg, err
 }
 
+// DEPRECATED: Replaced by GatewayConfig to centralize configuration for all gateway kinds
```

Comment:
Is there a reason to still leave this around? Isn't it internal to this package.

Reply:
Oh, I see. Its because the the names from the opaque config are different.